deploy/crds/infinispan.org_caches_crd.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: caches.infinispan.org
-  labels:
```

Comment:
@timothymathison please run `make codegen` for full code generation

Reply:
> @timothymathison please run `make codegen` for full code generation

Oddly, running `make crdsgen` (which is the first part of `make codegen`) is what removed this label. Is that desired?

Reply:
codegen will add required labels

Reply:
@dmvolod, after running `make codegen` the `label` is still removed.

Reply:
I'm not sure, maybe sed imp in crd-gen.sh works diffidence for Linux and Mac OS?

Reply:
@timothymathison could you please validate, it seems to this code doesn't work correctly on MacOS
```
sed -i -e "/name: infinispans.infinispan.org/a \  labels:\n    name: infinispan-operator" deploy/crds/infinispan.org_infinispans_crd.yaml
```

Reply:
@dmvolod Yes it was an issue with mac os `sed`. I fixed it by using `gsed` installed with `brew install gnu-sed`

Reply:
Have you try with this solution without additional software install
https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux

Reply:
@dmvolod the problem I was encountering was actually described here:
https://stackoverflow.com/questions/40843994/extra-characters-after-at-the-end-of-a-command

I wasn't able to find a nice solution without adding literal newlines. I can use that solution and test it in linux if you want me to.

Reply:
Yes, good point. I can test it on Linux, let me know if it needed.

Reply:
@dmvolod I attempted the fix described in that stack overflow. But that breaks it for gnu `sed`. I don't think there is a simple solution to make the same command work with both versions of `sed`.
I've left it the way I had it (where it checks if `gsed` is installed); unless we want to use something besides `sed`.

Reply:
Good, let's keep this solution. Thanks